app/assets/javascripts/controllers/container_topology/container_topology_controller.js

```diff
@@ -51,6 +58,92 @@ angular.module('topologyApp', ['kubernetesUI', 'ui.bootstrap'])
     $interval.cancel(promise);
   });
 
+
+  var refreshProvider = function(d) {
+    $scope.refreshProvider.value = true;
+    document.getElementsByClassName('alert alert-success alert-dismissable refresh_provider')[0].style.display ='block';
```

Comment:
@himdel , I added $timeout to the controller constructor, and then I added it per your recommendation to "wrap" the function on line 62
(var refreshProvider = $timeout(function(d) { ...)
and I commented out the "getElem..." part on line 64.
the green notification now never appears.